### PR TITLE
API Changes Fs Storage Documentation

### PR DESCRIPTION
**Pull request by amandawrcoelho**
Describe some inline functions

Signed-off-by: Amanda Coelho <amanda.winder.ribeiro.coelho@intel.com>

### CODE REVIEW

**Comment by amandawrcoelho** [comment]
#1758

File: src/lib/io/include/sol-fs-storage.h [hunk 1]
```diff
@@ -106,6 +106,15 @@ int sol_fs_read_raw(const char *name, struct sol_buffer *buffer);
         return -EINVAL; \
     }
 
+/**
+ * @brief Read an uint8_t in storage
```
Comment (cabelitos) [inline src/lib/io/include/sol-fs-storage.h]: from storage

File: src/lib/io/include/sol-fs-storage.h [hunk 2]
```diff
@@ -106,6 +106,15 @@ int sol_fs_read_raw(const char *name, struct sol_buffer *buffer);
...
+ *
+ * @param name name of property. It will look for a file on filesystem with
+ * this name
+ * @param value value sets the buffer that will be read
```
Comment (cabelitos) [inline src/lib/io/include/sol-fs-storage.h]: value is not a buffer.

Comment (amandawrcoelho) [inline src/lib/io/include/sol-fs-storage.h]: opppps

Comment (amandawrcoelho) [inline src/lib/io/include/sol-fs-storage.h]: What do you think about "@param value value helps to set a struct @ref sol_buffer"? @cabelitos

File: src/lib/io/include/sol-fs-storage.h (hunk 1 above)
Comment (amandawrcoelho) [inline src/lib/io/include/sol-fs-storage.h]: oppppppppppppps

**Pull request closed without merging**